sdk/cosmos/azure-cosmos-examples/src/test/java/com/azure/cosmos/rx/examples/Utils.java

```diff
@@ -5,9 +5,9 @@
 
 import com.azure.cosmos.ConnectionMode;
 import com.azure.cosmos.ConnectionPolicy;
-import com.azure.cosmos.FeedResponse;
-import com.azure.cosmos.ThrottlingRetryOptions;
-import com.azure.cosmos.SqlQuerySpec;
+import com.azure.cosmos.model.FeedResponse;
+import com.azure.cosmos.model.ThrottlingRetryOptions;
```

Comment:
question: ThrottingRetryOption is not a rest contract. it defines client behaviour on throttling. Is this qualify for a model?

Reply:
No it doesn't, as it specifies client configuration.